cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -854,3 +854,46 @@ def _revert():
     view = navigate_to(ansible_service, "Details")
     assert view.provisioning.credentials.get_text_of("Vault") == vault_creds.name
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+@pytest.mark.tier(3)
+@pytest.mark.meta(automates=[BZ(1734904)])
+def test_ansible_service_ansible_galaxy_role(
+    appliance,
+    request,
+    ansible_catalog_item,
+    ansible_service_catalog,
+    ansible_service_funcscope,
+    ansible_service_request_funcscope,
+):
+    """Check Role is fetched from Ansible Galaxy by using roles/requirements.yml file
+    from playbook.
+
+    Bugzilla:
+        1734904
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        initialEstimate: 1/3h
+        tags: ansible_embed
+    """
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {
+            "playbook": "ansible_galaxy_role_users.yaml"
+        }
+
+    @request.addfinalizer
+    def _revert():
+        with update(ansible_catalog_item):
+            ansible_catalog_item.provisioning = {"playbook": "dump_all_variables.yml"}
```

Comment:
please obtain this value from  catalog item before you update it.  if you change this value in ansible_catalog_item in the foreseeble future, this test will revert value to invalid value.

Reply:
@izapolsk I am resetting the value to the default, which is mention there in the `ansible_catalog_item` fixture for future use.

Reply:
Updated.